lib/browser/tag/tag.js

```diff
@@ -207,10 +207,12 @@ function Tag(impl, conf, innerHTML) {
           each(ptag.tags[tagName], function(tag, i) {
             if (tag._riot_id == self._riot_id)
               ptag.tags[tagName].splice(i, 1)
+            if (!ptag.tags[tagName].length) delete ptag.tags[tagName]
```

Comment:
It'd be clearer to refactor this into a removeChildTag method. I did something similar in #1525 while trying to solve a different issue.